project_resource_calendar/report/calendar_event_template.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="calendar_event_report_view">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="web.external_layout">
```

Comment:
Specs: format is missing. CSS class loaded on #216 you can reference to those i.e class="h3-report"

Reply:
@eilst Done